cfg.go

```diff
@@ -1,65 +1,35 @@
 package cfg
 
 import (
-	"flag"
-	"log"
-	"os"
-	"strings"
+	"fmt"
 
-	"github.com/joho/godotenv"
 	"github.com/spf13/viper"
 )
 
 const (
-	envModeVar      = "GOENV"
-	localConfigName = "local"
+	configType        = "yaml"
```

Comment:
`configType` must be open to changes

Reply:
I believe that unifying `configType` leads to more uniform code, but I'll restore to flexibility.